lib/braveAlerter.js

```diff
@@ -106,7 +112,7 @@ class BraveAlerter {
       }
     } catch (err) {
       // TODO Better error handling (maybe try to send the fallback message immediately)
-      helpers.logError(`Failed to send reminder message for session ${alertInfo.sessionId}. ${JSON.stringify(err)}`)
+      helpers.logError(`Failed to send reminder message for session ${alertInfo.sessionId}. ${err.toString()}`)
```

Comment:
Now that we have the Sentry messages, it's very obvious that this wasn't giving any useful information. Apparently when you `JSON.stringify` an Error object, it just outputs the string `{}`, which is USELESS! As opposed to the Error's `toString()` function which outputs much more useful debugging information.